scst-store/configure-access-token.hbs.md

```diff
@@ -0,0 +1,16 @@
+# Configure Access Tokens
+
+Service accounts are required to generate the access tokens.  
+
+The access token is a `Bearer` token used in the http request header `Authorization`. (ex. `Authorization: Bearer eyJhbGciOiJSUzI1NiIsImtpZCI6IjhMV0...`)
+
+By default, Supply Chain Security Tools - Store includes `read-write` service account installed.
+This service account is cluster-wide.
+
+## Setting the Access Token
+
+{{> 'partials/insight-set-access-token' }}
+
+## Additional Resources
+
+For instructions on creating your own service accounts, see [Create Service Accounts](create-service-account.hbs.md).
```

Comment:
Good callout, but now that I look at it I think that this can be moved right under the intro sentence. Reframe it as a prerequisite. And combine it with the existing sentence

> By default, Supply Chain Security Tools - Store includes `read-write` service account installed.
This service account is cluster-wide.